client/signup/config/steps-pure.js

```diff
@@ -298,6 +298,17 @@ export function generateSteps( {
 		},
 
 		'plans-business': {
+			stepName: 'plans-business',
+			apiRequestFunction: addPlanToCart,
+			fulfilledStepCallback: isPlanFulfilled,
+			dependencies: [ 'siteSlug' ],
+			providesDependencies: [ 'cartItem' ],
+			defaultDependencies: {
+				cartItem: PLAN_BUSINESS,
+			},
+		},
+
+		'plans-business-with-plugin': {
 			stepName: 'plans-business',
```

Comment:
The `stepName` needs fixing here.

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/80105/commits/ced8e07ecf435c12b717213252fb444474bc961f